vault-admin-client/src/test/java/com/scality/osis/vaultadmin/impl/VaultAdminImplTest.java

```diff
@@ -208,7 +211,7 @@ public void listAccounts500ServiceResponse() {
     @Test
     public void testVaultAdminImplInvalidEndpoint() {
         assertThrows(IllegalArgumentException.class,
-                () -> new VaultAdminImpl(accountServicesClient, "dummy_endpoint"),
+                () -> new VaultAdminImpl(accountServicesClient, stsClient, "dummy_endpoint", s3Endpoint),
```

Comment:
Same here. May be Vault endpoint.

Reply:
Done in 9101950281bb1e469ee5c3cbe333dcaa4c9a3953